### DISCUSSION TITLE
Mocking `WagmiConfig` for unit testing purposes

### DISCUSSION THREAD

**Discussion by firatkaratas** [wevm/wagmi#537] [Q&A]
Hey all,

I would like to test one of my custom hooks that uses `useAccount()`. When I'm trying to run the test ( with 'Jest' ), I have been given the error: 

`No QueryClient set, use QueryClientProvider to set one` from the line of `useAccount`.  

I assume this is happening because I haven't provided the `WagmiConfig` as a context in my unit test. How can I mock the `WagmiConfig` for testing purposes? 

Thanks in advance!

**Comment by tmm** [comment]
Check out how we [test the `react` package](https://github.com/tmm/wagmi/blob/main/packages/react/test/index.tsx).
